tests/skipped_tests.tbl

```diff
@@ -430,6 +430,9 @@ tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_
 tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_mixed_start_stop
 tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_mixed_start_stop2
 tests/third_party/cupy/creation_tests/test_ranges.py::TestRanges::test_linspace_start_stop_list
+tests/third_party/cupy/creation_tests/test_matrix.py::TestTriLowerAndUpper_param_0_{shape=(2,)}::test_tril
```

Comment:
why do you mark the tests as to be skipped?

Reply:
dpctl.tensor.tril() does not support one dimension array.
Array API standard definition:
x (array) – input array having shape (..., M, N) and whose innermost two dimensions form MxN matrices.

Reply:
can you add a fixture:
```
@pytest.mark.usefixtures("allow_fall_back_on_numpy")
```
before
```
class TestTriLowerAndUpper(unittest.TestCase):
```
to allow test to fallback on numpy with shape=(2,) rather then to silently skip it.

Any nested "allow_fall_back_on_numpy" fixtures above class methods can be removed then.